site/content/en/reference/live/alpha.md

```diff
@@ -0,0 +1,156 @@
+---
+title: "Inventory Object Upgrade (alpha)"
+linkTitle: "inventory object upgrade (alpha)"
+weight: 3
+type: docs
+description: >
+   Instructions (alpha) to upgrade to next generation inventory object
+---
+<!--mdtogo:Short
+    Instructions (alpha) to upgrade to next generation inventory object
+-->
+
+### How to Upgrade to the Next Generation Inventory Object (Alpha)
+
+#### What is an Inventory Object
+
+An inventory object is the automatically generated object which keeps track
+of the set of objects applied together. The current inventory object type
+is a [**ConfigMap**](https://kubernetes.io/docs/concepts/configuration/configmap/),
+and it is usually defined in a package file called
+**inventory-template.yaml**. This file is created from an invocation of
+`kpt live init`. A typical use of the inventory object is to prune (delete)
+objects omitted locally.
+
+#### What is a Next Generation Inventory Object
+
+The next generation inventory object is a **ResourceGroup** custom resource
+replacing the current [**ConfigMap**](https://kubernetes.io/docs/concepts/configuration/configmap/).
+Because the new inventory object is a
+custom resource, you must have permissions to add a custom resource
+definition (CRD) to the cluster. The individual command to add the
+**ResourceGroup** CRD is:
+
+```sh
+export RESOURCE_GROUP_INVENTORY=1
+kpt live install-resource-group
```

Comment:
We discussed that in a normal flow, `kpt live init` automatically attempts to install the CRD, and failing that, this is the manual fallback, right? In the case, then you can just show the happy path and have this as an aside.

Reply:
OK. I removed this example to the `kpt live install-resource-group`. I added an explanation further down that the CRD is normally added as an initial step during `kpt live apply`. Please let me know what you think.